lib/events/sessionlog.go

```diff
@@ -56,11 +56,10 @@ func chunksFileName(dataDir string, sessionID session.ID, offset int64) string {
 }
 
 type indexEntry struct {
-	FileName   string `json:"file_name"`
-	Type       string `json:"type"`
-	Index      int64  `json:"index"`
-	Offset     int64  `json:"offset,"`
-	authServer string
+	FileName string `json:"file_name"`
+	Type     string `json:"type"`
+	Index    int64  `json:"index"`
+	Offset   int64  `json:"offset,"`
```

Comment:
```suggestion
	Offset   int64  `json:"offset"`
```

I wonder if this has actually caused any issues in the past..

Reply:
Hmm - any potential risk from changing it now ? Unsure if this type is persisted anywhere.

Reply:
Turns out we can just delete this if we delete SSHPlaybackWriter 🚀